examples/gl/billboardExample/bin/data/shadersGLES3/Billboard.frag

```diff
@@ -0,0 +1,13 @@
+#version 300 es
+
```

Comment:
Hey @Jonathhhan - Is this the only change needed in the shader?   
If so I might look at a core/code solution

Reply:
@ofTheo yes, and `precision lowp float;` in the fragment shader.

Reply:
Ahh okay - that does make it a bit trickier. 
Was thinking we could just do a header replace at the shader level for emscripten. 

the float thing is tricky as it prob isn't good to hide the precision from the user ( ie: replace `float` with `precision lowp  float'` for emscripten automatically )